src/components/Modal/__test__/modal.spec.js

```diff
@@ -187,4 +187,24 @@ describe('<Modal/>', () => {
         component.unmount();
         expect(CounterManager.decrement).not.toHaveBeenCalled();
     });
+    it('should show the cross for close the modal by default', () => {
+        const component = mount(
+            <Modal isOpen>
+                <p />
+            </Modal>,
+        );
+
+        expect(component.find('button[data-id="button-icon-element"]').prop('id')).toBe(
+            'modal-close-button',
+        );
```

Comment:
you can simply:
```
import StyledCloseButton from '../styled/closeButton';
```
and then:
```expect(component.find(StyledCloseButton).exists()).toBe(true);```
same for the case below since ```button[data-id="button-icon-element"]``` is an internal implementation of ButtonIcon and this implies an introspection that we need to avoid.

Reply:
I have added this suggestion, thank you very much